README.md

```diff
@@ -55,128 +58,75 @@ package:
 - [Pip]
 - [CMake] version 3.5 or higher (e.g., `pip3 install cmake`)
 - Build system: [Make] or [Ninja]
-- for developers: Cython version 0.21 or higher (e.g., `pip3 install cython`)
-- Sphinx and it's bootstrap theme (e.g., `pip3 install sphinx sphinx_bootstrap_theme`)
-
-## Installation instructions
-
-### Installing NetworKit via pip
-
-    pip3 install [--user] networkit
-
-Depending on your system, you might need to install python3-tk.
-
-
-### Building NetworKit as a Python Module from source
-
-Run the script `setup.py` with the following options:
-
-	python3 setup.py build_ext [-jX]
-
-The script will call `cmake` and `ninja` (`make` as fallback) to compile NetworKit as a library, build the extensions and copy it to the top folder. By default, NetworKit will be built with the amount of available cores in optimized mode. It is possible the add the option `-jN` the number of threads used for compilation.
-The setup script provides more functionality and can be used with pip as well:
-
-	pip3 install -e .
-
-will compile NetworKit, build the extensions and on top of that temporarily install NetworKit so that it is available on the whole system. `pip3 uninstall networkit` will remove networkit.
-
-### `jupyterhub`: An interactive environment for working with NetworKit
-
-To check that everything works as expected, open a python terminal and run the example from the beginning of this document or the following lines:
-
-    >>> import networkit
-    >>> G = networkit.Graph(5)
-    >>> G.addEdge(0,1)
-    >>> G.toString()
-
-Additionally, we recommend that you familiarize yourself with NetworKit through experimenting with the interactive IPython Notebook `NetworKit_UserGuide.ipynb` located in the folder `Doc/Notebooks`. The user guide also introduces a large portion of NetworKit functionality with usage examples. To display and work with these notebooks, you have to install jupyterhub and start a local notebook server from the terminal with:
+- Cython version 0.21 or higher (e.g., `pip3 install cython`)
 
-	jupyterhub --no-ssl
+# Usage example
 
-If you run into any problems with jupyterhub, head over to the [jupyterhub documentation](https://jupyterhub.readthedocs.io/en/latest/) and make sure, you have the listed packages installed. If the notebook server starts as it is supposed to, your default browser should open a web interface or you have to open it manually. Then you can add `NetworKit_UserGuide.ipynb` from the above mentioned location or browse to the location through the web interface.
+In this example we generate a random hyperbolic graph with 100k nodes
+and we compute its communities with the PLM method:
```

Comment:
We should point users to our notebooks here! (I am not even sure if we want to keep the small example -- the notebooks are much more instructive.)

Reply:
+1 for pointing to the notebooks. 

If shortening the readme is intended - than maybe also remove the example. However: Although the notebooks are much more instructive, their results don't show when viewed in the web on git. Maybe it's good for first-time users to see how an output should look like?

Reply:
I think it is better to keep a short example to show basic APIs. Anyway, we should definitely add a link to the notebooks here.

Reply:
@fabratu can you push another commit with the links?

Reply:
Only partially related - but since there is now more than one notebook ... is there some kind of order in how to read them? If so, should we include a prefix in the file-naming? And furthermore: does User-Guide should still be named that way or more like: Overview, Introduction?

Reply:
Moved / Edited the notebook ref.